Scenes/Tavern/TavernScreen.gd

```diff
@@ -9,6 +9,7 @@ const MENU_PATH = "res://Scenes/MainMenu/MainMenu.tscn"
 onready var game_manager = get_parent().find_node("GameManager")
 onready var dwarves_manager = get_parent().find_node("DwarvesManager")
 onready var devil_spawner = get_node("/root/World").find_node("DevilSpawner")
+onready var elf = get_node("/root/World/MainObjectsLayer/Elf")
```

Comment:
Better use `find_node` method, to avoid conflicts when changing directory in future.

```suggestion
onready var elf = get_node("/root/World").find_node("Elf")
```